swagger.yml

```diff
@@ -1629,6 +1630,160 @@ paths:
         default:
           $ref: '#/responses/error'
   # end site/split_testing/{split_test_id/unpublish}
+  # begin /dns_zones/
+  /dns_zones:
+    post:
+      operationId: createDnsZone
+      consumes:
+        - application/json
+      parameters:
+        - name: DnsZoneParams
+          in: body
+          required: true
+          schema:
+            $ref: '#/definitions/dnsZoneSetup'
+      responses:
+        '201':
+          description: Created
+          schema:
+            $ref: '#/definitions/dnsZone'
+        default:
+          $ref: '#/responses/error'
+    get:
+      operationId: getDnsZones
+      tags: [dnsZone]
+      responses:
+        '200':
+          description: 'get all DNS zones your user account has access to'
```

Comment:
by default (if you don't pass anything), this endpoint will return all zones that the user has access to. if you pass `account_slug` as a param, it'll return all zones that the account has access to.
it would be nice if we could tweak a description a bit, as "user account" is a bit confusing (Netlify has the concept of "user" and "account" in open-api).
how about changing it to either "get all DNS zones the user has access to" (drop the account), or add `account_slug` as an optional param here and say "get all DNS zones the user or account has access to`?

Reply:
I'll change the description.

About the optional parameter, I assume it is it a query parameter like this:
`/dns_zones?account_slug=xyz`

And not a parameter in the path like the members and sites endpoints:
`/{account_slug}/sites`
`/{account_slug}/members`

Reply:
> About the optional parameter, I assume it is it a query parameter like this:
`/dns_zones?account_slug=xyz`

yes, exactly!